.github/CODEOWNERS

```diff
@@ -2,10 +2,9 @@
 /cue @AntoineThebaud
 /docs @AntoineThebaud
 /go-sdk @Gladorme
-/internal @Nexucis
-/internal/api/impl/auth @Nexucis @celian-garcia
+/internal @Nexucis @celian-garcia
 /internal/api/plugin/migrate @Nexucis @AntoineThebaud
 /internal/api/plugin/schema @Nexucis @AntoineThebaud
-/pkg/client @Nexucis @celian-garcia
+/pkg @Nexucis @celian-garcia
```

Comment:
I don't think it will work as intended. Meaning that  for `/pkg/model` Celian will be owner as there is a dedicated rule for that.

And since there is only two sub package for `/pkg`, that means Celian is actually owner of `pkg/client`

Reply:
Ok your point is valid, I'll revert to the previous state then.